### PR TITLE
Add nr1-cambio

### PR DESCRIPTION
**Pull request by YeisonGaona**

### CODE REVIEW

**Comment by jpvajda** [comment]
@YeisonGaona do you mind providing a screen shot of this rendering locally, so we can validate how the page looks?

**Review by jpvajda — COMMENTED** [review]
please provide the screenshot of the local render of this page. - thanks!

**Comment by YeisonGaona** [comment]
> please provide the screenshot of the local render of this page. - thanks!

@jpvajda This is the screenshot. I found a problem with some capitalization, but corrected it with a third commit.

image

File: src/data/project-main-content/globalsoftWigilabs-nr1-cambio.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+date: "02/15/2021"
+title: "New Relic One Cambio"
+projectConfig: "src/data/projects/globalsoftwigilabs-nr1-cambio.json"
+---
+
+## Getting Started
+
+Go to the project's [README](https://github.com/GlobalsoftWigilabs/nr1-cambio#readme) for setup and usage details.
+
+<!--
```
Comment (jpvajda) [inline src/data/project-main-content/globalsoftWigilabs-nr1-cambio.mdx:12]: Can you remove this commented code?

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: src/data/project-main-content/globalsoftWigilabs-nr1-cambio.mdx (hunk 1 above)
Comment (YeisonGaona) [reply src/data/project-main-content/globalsoftWigilabs-nr1-cambio.mdx]: > Can you remove this commented code?

It's done

File: src/data/projects/globalsoftWigilabs-nr1-cambio.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "description": "This is a New Relic One Application that allows you to connect to the Datadog API and show you important information such as your Dashboards, Alerts, integrations, and much more.",
+    "ossCategory": "new-relic-experimental",
+    "primaryLanguage": "JavaScript",
+    "projectTags": ["nr1-app"],
+    "acceptsContributions": true,
+    "website": {
+      "title": "Cambio ",
+      "url": "https://github.com/GlobalsoftWigilabs/nr1-cambio"
+    }
+  }
```
Comment (jpvajda) [inline src/data/projects/globalsoftWigilabs-nr1-cambio.json:25]:
```suggestion
  }
  
```

Comment (jpvajda) [reply src/data/projects/globalsoftWigilabs-nr1-cambio.json:25]: added a new line to remove the no line at end of file marker

**Review by jpvajda — APPROVED** [review]

**Comment by jpvajda** [comment]
@YeisonGaona there is actually one more change here, in order for this to automatically bump our release version you'll need to use conventional commits, can you go and change your most recent commit message following this format:

https://github.com/newrelic/opensource-website/blob/develop/.github/pull_request_template.md

I recommend using git rebase -i 

https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history

**Review by jpvajda — COMMENTED** [review]
needs a conventional commit message.

**Comment by YeisonGaona** [comment]
> needs a conventional commit message.

It's done @jpvajda

**Comment by jpvajda** [comment]
@YeisonGaona Thank you so much for working through this review with me and contributing! 🥳

**Comment by YeisonGaona** [comment]
Thanks @jpvajda for you time !

**Pull request merged**